cms/search/tests/test_api_views.py

```diff
@@ -0,0 +1,138 @@
+
+    def test_resources_returns_200_and_lists_various_page_types(self):
+        """Endpoint should return 200 and include all included_pages in the items."""
+        response = self.call_view_as_external(RESOURCE_ENDPOINT)
+        self.assertEqual(response.status_code, 200, f"Expected 200 OK from {RESOURCE_ENDPOINT}")
+
+        data = self.parse_json(response)
+        self.assertIn("items", data, "Response JSON should contain 'items' list")
+
+        for page in self.included_pages:
+            matching = self.get_page_dict(data, page)
+            self.assertIsNotNone(matching, f"Expected page with URI {page.url_path} to be present in the items")
+            self.assert_base_fields(matching, page)
+
+    def test_resources_excludes_non_indexable_pages(self):
+        """Non-indexable pages (ArticleSeries, Home, ReleaseCalendarIndex, Theme, Topic)
+        should not appear in items.
+        """
+        response = self.call_view_as_external(RESOURCE_ENDPOINT)
+        self.assertEqual(response.status_code, 200)
+
+        data = self.parse_json(response)
+        for page in self.excluded_pages:
+            self.assertIsNone(
+                self.get_page_dict(data, page), f"Expected page with URI {page.url_path} not to be present"
+            )
+
+    def test_resources_disabled_when_external_env_false(self):
+        """Endpoint should return 404 when IS_EXTERNAL_ENV=False."""
+        response = self.client.get(RESOURCE_ENDPOINT)
+        self.assertEqual(response.status_code, 404)
+
+
+@override_settings(IS_EXTERNAL_ENV=False)
+class ResourceListViewPaginationTests(TestCase, ExternalAPITestMixin):
+    @classmethod
+    def setUpTestData(cls):
+        cls.index_page = IndexPageFactory(slug="custom-slug-1")
+        # One parent + 12 child pages = 13 total
+        cls.pages = InformationPageFactory.create_batch(
+            12, parent=cls.index_page, slug=factory.Sequence(lambda n: f"test_page_{n + 1}")
+        )
+        cls.total_resources = len(cls.pages) + 1  # +1 for the parent
+
+    def test_default_pagination_returns_first_slice(self):
+        """With no limit/ offset specified we should get DEFAULT_LIMIT (or all if fewer)."""
+        response = self.call_view_as_external("/v1/resources/")
+        self.assertEqual(response.status_code, 200)
+
+        data = self.parse_json(response)
+
+        expected_items = min(
+            self.total_resources, settings.SEARCH_API_DEFAULT_PAGE_SIZE
+        )  # Default page size is 20 currently
+        self.assertEqual(len(data["items"]), expected_items)
+        self.assertEqual(data["count"], expected_items)
+        self.assertEqual(data["total_count"], self.total_resources)
+        self.assertEqual(data["offset"], 0)
+
+    def test_second_slice_returns_remaining_items(self):
+        """Requesting offset=<7> should return whatever is left after the first slice."""
+        response = self.call_view_as_external(f"/v1/resources/?offset={7}")
```

Comment:
Do we need an f-string here? Here and on line 131. I see a mixture of approaches (e.g. line 118)

```suggestion
        response = self.call_view_as_external("/v1/resources/?offset=7")
```

Reply:
Done!

I have a `RESOURCE_ENDPOINT = "/v1/resources/"` constant and I now utilise that consistently in the whole file =)

Reply:
Thanks, do we still need brackets for the integers though? Is that for readability?

Reply:
It was initally because it was _readable/ stuck out to me_ but doing some quick google search on the best practices, standards, and the pythonic way. `{}` in f-strings shouldn't be used for this and this is not idiomatic.

So went with the style of `response = self.call_view_as_external(f"{RESOURCE_ENDPOINT}?offset=7")` etc